uber/templates/preregistration/preregbase.html

```diff
@@ -95,7 +101,9 @@
                 price: {{ c.BADGE_TYPE_PRICES[badge_type] }},
                 badge_type: '{{ badge_type }}',
                 onClick: function () {
-                    $('input[name="badge_type"]').val('{{ badge_type }}');
+                    if ($('input[name="badge_type"]').val() != '{{ c.STAFF_BADGE }}') {
```

Comment:
Are you sure about that? Removing a staff badge from someone has a bunch of knock-on effects. Effects that an attendee will certainly not be aware of when they are editing their info.